generate-release-notes/README.md

```diff
@@ -0,0 +1,84 @@
+# OpenJDK Release Notes Tool
+
+This repository contains two scripts that can be used to generate OpenJDK notes. Typical usage will be to first generate the list of commits for the release using the `fetchCommitList.js` script and then run `fetchReleaseNotes.js` script, supplying the commit list as an input.
```

Comment:
```suggestion
This repository contains two scripts that can be used to generate OpenJDK release notes. Typical usage will be to first generate the list of commits for the release using the `fetchCommitList.js` script and then run the `fetchReleaseNotes.js` script, supplying the commit list from `fetchCommitList.js` as an input.
```

Reply:
It was intentional to provide some example output to share. I can delete (or reduce) if it is preferred.